File: colegiocuotas/src/main/java/com/colegio/opticos/colegiocuotas/dto/MatriculadoDTO.java
package com.colegio.opticos.colegiocuotas.dto;

import lombok.*;
import java.time.LocalDate;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class MatriculadoDTO {
	private Long id;
    private String nombre;
    private Integer matricula;
    private String dni;
    private String telefono;
    private LocalDate fechaRegistro;
    private String email;
    private boolean habilitado;
    private boolean pagoAprobado;
    private int cuotasImpagas;
}

File: colegio-opticos/src/app/models/matriculadoDTO.ts
export interface MatriculadoDTO {
    id: number;
    nombre: string;
    username: string;
    matricula: number;
    dni: string;
    telefono: string;
    fechaRegistro: string;
    email: string;
    habilitado: boolean;
    cuotasImpagas: number;
    pagoAprobado: boolean;
  }